lib/ansible/modules/windows/win_chocolatey_feature.ps1

```diff
@@ -0,0 +1,75 @@
+#!powershell
+# This file is part of Ansible
+
+# Copyright (c) 2018 Ansible Project
```

Comment:
Another minor nitpick is that the syntax for copyright lines used to be:
```python
# Copyright: (c) 2018, Ansible Project
```

I know people have been adding other stuff since, but this was originally documented and automated by @abadger and I have been updating at least my modules like this.

Reply:
BTW I see you are doing this for the .py file.